blueocean-core-js/src/js/config.js

```diff
@@ -15,6 +15,8 @@ export default {
             // Look up where the Blue Ocean app is hosted
             config.blueoceanAppURL = headElement.getAttribute('data-appurl');
 
+            // load server skewTime
+            config.serverBrowserTimeSkewMillis = headElement.getAttribute('data-servertime') - Date.now();
```

Comment:
yes, but IMO we should move it to here. This way we can drop the `this.context.config` usage